src/services/AlertService.ts

```diff
@@ -49,7 +49,7 @@ export class AlertService {
   }
 
   public showAlert(config: IAlertModel): Promise<DialogCloseResult> {
-    return this.dialogService.open(Alert, config, { keyboard: true });
+    return this.dialogService.open(Alert, config, { keyboard: true }, config.className);
```

Comment:
I am concerned that this className is replacing pPopupModal.  This is potentially breaking the system.

I think that `config.className` would best not be passed in this way, rather should continue through in the model to pPopup-Modal where it can be applied at the top of pPopup-Modal.html

Reply:
Should we not hardcode the "crucial part for the system" somewhere then?
```ts
className = "pPopupModal", // gets set on ux-dialog-container
```
Ie. move class `pPopupModal` some levels below
![image](https://user-images.githubusercontent.com/30693990/157407367-6a4ff49f-b435-4151-9f93-ea4c79c23ccf.png)

Also, a search for `pPopupModal` is not used anywhere
![image](https://user-images.githubusercontent.com/30693990/157406322-5b58f8d8-7a6f-49f3-af1b-eb7e912720d0.png)

Reply:
Currently `pPopupModal` is set in 2 places
![image](https://user-images.githubusercontent.com/30693990/157407729-8224e2eb-7b90-48a0-a1f5-7587113ad449.png)

Imo, the top one should be customizable (as done in this case).
The crucial part is handled in the red circle below